### ISSUE TITLE
How to train my own dataset?

### ISSUE THREAD

**Issue by LaFeuilleMorte** [kcheng1021/GaussianPro#71]
Hi, I've tried using your training script to train my own dataset. My own dataset was preprocessed with colmap. And it's downsampled from a video which makes the images sequential. But when I tried to run your training code. The loss seemed to be very huge. And the loss won't decrease from the step 7000, And the final PSNR was 5dB, I doubt if there's a bug inside your code.

My training script is:
DATASET_DIR=****
SAVE_PATH=****

python train.py -s $DATASET_DIR -m $SAVE_PATH \
                --eval --flatten_loss --position_lr_init 0.000016 --scaling_lr 0.001 --percent_dense 0.0005 --port 1021 \
                --normal_loss --depth_loss --propagation_interval 50 --depth_error_min_threshold 0.8 --depth_error_max_threshold 1.0 \
                --propagated_iteration_begin 1000 --propagated_iteration_after 6000 --patch_size 20 --lambda_l1_normal 0.001 --lambda_cos_normal 0.001 \
                --data_device cuda --resolution 1
